lib/MusicBrainz/Server/Edit/Release/Merge.pm

```diff
@@ -219,6 +219,11 @@ override build_display_data => sub
                 mediums => $_->{mediums}
             }, @{ $self->data->{medium_changes} }
         ];
+        $data->{empty_releases} = [
+            map +{
+                release => $loaded->{Release}{ $_->{id} }
```

Comment:
I'd add at least
```suggestion
                release => $loaded->{Release}{ $_->{id} } // Release->new(name => $_->{name}),
```
here so that it displays the old name rather than `[removed]` once the merge succeeds. (The edit data contains more information that just the name we could display, though we'd have to build all the Moose entities here ourselves, and it already only uses the name for the non-empty releases, so I guess it's more of a future improvement.)